internal/resource/config/authenticationpolicies/settings/authentication_policies_settings_data_source.go

```diff
@@ -0,0 +1,95 @@
+package authenticationpoliciessettings
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	client "github.com/pingidentity/pingfederate-go-client/v1125/configurationapi"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest/common/pointers"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/datasource/common/id"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/resource/config"
+	internaltypes "github.com/pingidentity/terraform-provider-pingfederate/internal/types"
+)
+
+// Ensure the implementation satisfies the expected interfaces.
+var (
+	_ datasource.DataSource              = &authenticationPoliciesSettingsDataSource{}
+	_ datasource.DataSourceWithConfigure = &authenticationPoliciesSettingsDataSource{}
+)
+
+// AuthenticationPoliciesSettingsDataSource is a helper function to simplify the provider implementation.
+func AuthenticationPoliciesSettingsDataSource() datasource.DataSource {
+	return &authenticationPoliciesSettingsDataSource{}
+}
+
+// authenticationPoliciesSettingsDataSource is the datasource implementation.
+type authenticationPoliciesSettingsDataSource struct {
+	providerConfig internaltypes.ProviderConfiguration
+	apiClient      *client.APIClient
+}
+
+// GetSchema defines the schema for the datasource.
+func (r *authenticationPoliciesSettingsDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	schema := schema.Schema{
+		Description: "Describes the authentication policies settings",
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
```

Comment:
The id.ToDataSourceSchema call below adds `id` to the schema, so we don't need to duplicate it here